PHP/Language.md

```diff
@@ -263,7 +292,7 @@ Specifying a class with bindTo()
 
 Predefined Interfaces:  ArrayAccess, Serializable, Traversable, Iterator and IteratorAggregate
 
-Generators
+Generators. yield, yield from, gerReturn()
```

Comment:
::getReturn

Reply:
I did not add `::` at first to not mix up with static methods. But for showing that it **is** a method, I'll add it.